src/app.jsx

```diff
@@ -60,6 +63,18 @@ export class Application extends React.Component {
         this.setState({ alerts: alerts });
     }
 
+    checkExpiration(cert) {
+        const eventdate = moment(Number(cert["not-valid-after"].v) * 1000);
+        const todaysdate = moment();
+        const diffDays = eventdate.diff(todaysdate, "days");
+        const diffSeconds = eventdate.diff(todaysdate, "days");
```

Comment:
These two variables are the same. It seems like a copy-paste error to me.

Reply:
Fixed